.github/workflows/ci.yml

```diff
@@ -14,34 +14,3 @@ jobs:
     - name: Run presubmit checks
       run: |
         make presubmit
-
```

Comment:
maybe call this file presubmit.yml as CI name does not make sense. all stuff in this dir is CI ?

Reply:
Done.